edx_reverification_block/xblock/reverification_block.py

```diff
@@ -65,6 +68,13 @@ class ReverificationBlock(XBlock):
         help="ISO-8601 formatted string representing the due date of this related assessment."
     )
 
+    grace_period = Integer(
+        display_name="Verification Grace Period",
+        scope=Scope.settings,
+        default=ICRV_GRACE_PERIOD,
+        help="The number of days learners can verify their identity after the exam's due date."
```

Comment:
@awais786 @srpearce There is nothing limiting ICRV to exams. I suggest "exam" be replaced with "assignment".